test/inductor/test_metrics.py

```diff
@@ -91,6 +92,7 @@ def f(lhs, index, rhs):
         kernel_code = kernel_list[0]
         self.assertEqual(metrics._count_pattern(kernel_code, "tl.atomic_add"), 1)
 
+    @largeTensorTest(25e7 * 2 * 4, device=GPU_TYPE)
```

Comment:
Fair enough, but how come this was passing before? Or it was not, and it was just not being tested on small GPU boxes?

Reply:
It's only actually 2 GB of data, but was failing for me because of other users on the machine. Would be fine in CI.